js/liquid.js

```diff
@@ -1072,14 +1072,7 @@ module.exports = class liquid extends Exchange {
         const price = this.safeNumber (order, 'price');
```

Comment:
The price, amount, filled and other monetary values are also fed to safeOrder as strings.

Reply:
right! i didn't look into the old implementation fully, had to check, my mistake.